tests/modules/programs/pandoc/templates.nix

```
@@ -0,0 +1,20 @@
+{ config, ... }:
+
+let cfg = config.programs.pandoc;
+
+in {
+  config = {
+    programs.pandoc = {
+      enable = true;
+
+      templates = { "default.latex" = ./template.latex; };
+    };
```

To avoid downloading Pandoc unnecessarily:
```suggestion
    };
    
    test.stubs.pandoc = { };
```